staff/nicolas-martin/e-commerce-protos/scripts/cart.test.js

```diff
@@ -0,0 +1,137 @@
+function createSetOfProducts() {
+  var cart = new Cart;
+
+  var tshirt = new TShirt(120, "Supreme", "white", "M");
+  cart.add(tshirt);
+  var sweater = new Sweater(45, "Diesel", "black", "M");
+  cart.add(sweater);
+  var laptop = new Laptop(33, "Apple", "MacBook Pro", 15);
+  cart.add(laptop);
+  var desktop = new Desktop(9.99, "HP", "1800", 20);
+  cart.add(desktop);
+  var mobile = new Mobile(12, "Apple", "iPhone X", "space-gray");
+  cart.add(mobile);
+  var mobile2 = new Mobile(12.22, "Xiaomi", "5X", "space-rose");
+  cart.add(mobile2);
+
+  return cart;
+}
+
+suite("cart", function(){
+
+    describe('*** manage products from Cart ***', function(){
```

Comment:
I would make one describe for each method of the cart and inside each describe I would implement all tests of that method

Reply:
done